packages/create/templates/hello/src/operations.ts

```diff
@@ -11,13 +11,13 @@ export class Hello {
 
   @GetApi('/') // Serve a quick readme for the app
   static async readme(_ctxt: HandlerContext) {
-    const readme = '<html><body><p>' +
-           'Welcome to the DBOS Hello App!<br><br>' +
-           'Visit the route /greeting/:name to be greeted!<br>' +
-           'For example, visit <a href="/greeting/dbos">/greeting/dbos</a>.<br>' +
-           'The counter increments with each page visit.<br>' +
-           'If you visit a new name like <a href="/greeting/alice">/greeting/alice</a>, the counter starts at 1.' +
-           '</p></body></html>';
+    const readme = `<html><body><p>
+           Welcome to the DBOS Hello App!<br><br>
+           Visit the route /greeting/:name to be greeted!<br>' +
+           For example, visit <a href="/greeting/dbos">/greeting/dbos</a>.<br>
+           The counter increments with each page visit.<br>
+           If you visit a new name like <a href="/greeting/alice">/greeting/alice</a>, the counter starts at 1.
+           </p></body></html>`;
```

Comment:
Oh yea that's neat :)

Reply:
@apoliakov I did suggest this in your original [PR](https://github.com/dbos-inc/dbos-transact/pull/555#discussion_r1694530812) but it was buried by other stuff :p